Source/core/webcl/WebCLException.cpp

```diff
@@ -7,9 +7,6 @@
 
 #if ENABLE(WEBCL)
 
-#include "modules/webcl/WebCL.h"
```

Comment:
I think this is for the definition of CL_* in this file, though WebCL.h is not the exact file that should be included here. The original header inclusion is a little messy, so I'd like to fix that later.